src/receive_imf.rs

```diff
@@ -1550,6 +1551,10 @@ async fn add_parts(
         let part_is_empty =
             typ == Viewtype::Text && msg.is_empty() && part.param.get(Param::Quote).is_none();
 
+        if let Some(contact_id) = group_changes.added_removed_id {
+            param.set_int(Param::ContactAddedRemoved, contact_id.to_u32() as i32);
```

Comment:
`as` casts may panic, and since panics are super hard to debug, we try to avoid them where possible.

```suggestion
            param.set(Param::ContactAddedRemoved, contact_id.to_u32().to_string());
```